Dockerfile

```diff
@@ -11,7 +11,8 @@ ENV PYTHONPATH /app:/app/coda
 RUN mkdir /app
 WORKDIR /app
 
-RUN apt-get update -qq && apt-get install -y mysql-client netcat
+RUN apt-get update -qq && \
+    apt-get install -y  mariadb-server mariadb-client libmariadbclient-dev libssl-dev netcat
```

Comment:
So did you try without changing this line? I don't think we needed to change the client, mysql-client should work just fine connecting to mariadb.

Reply:
you are right! `mysql-client` works just fine. I will revert it to what it was.